### DISCUSSION TITLE
2021-11-17 Weekly Meeting 30 Agenda / Notes

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#125] [Weekly Meetings]
Weekly Meeting 30 is happening on Wednesday November 17th, 2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
   - Welcome new members and attendance
   - Backlog refinement
     - https://github.com/omigroup/omigroup/issues/72
     - #118 Add vision to readme (see PR #120) [DONE]
     - [website](https://github.com/omigroup/website/projects/1)
     - [Call for participation: Metaversity](https://github.com/omigroup/omigroup/issues/115) [DONE]
     - [Call for participation: XR Safety Week](https://github.com/omigroup/omigroup/issues/124)
     - [Suggestion from Thiizic of Futurist Foundation: Partner Program](https://github.com/omigroup/omigroup/issues/121)
     - https://github.com/omigroup/omigroup/issues/109
   - [glTF subgroup](https://github.com/omigroup/gltf-extensions/discussions/31)
     - Khronos Liaison 
   - WebXR Hackathon - "Immersive Web Fest"
   - VKet Booth

 ## Add Ons
- Twitter Spaces with Evo / Jin
  - [Jin](https://twitter.com/dankvr) hosting the first one was on Friday at 2PM ET
- Pick up OMI Chan work this week?
- [Avacon](https://www.avacon.org/blog/)
- next week: leveraging roles that match up with working groups

**Comment by mrmetaverse** [comment]
Added new items from our backlog

**Comment by mrmetaverse** [comment]
Mapping the Metaverse group will meet Tuesdays at Noon PST (-8 UTC) 
Jin is also running M3 events on Friday at 11am PT / 2PM ET

**Comment by mrmetaverse** [comment]
Website working group needs to find a new time to meet 
#38

**Comment by mrmetaverse** [comment]
gLTF working group notes from last week: 

https://github.com/omigroup/gltf-extensions/discussions/33

**Comment by mrmetaverse** [comment]
WebXR Hackathon has a new name: "Immersive Web Fest" 

@msub2 is still organizing, and has ideas of how to break up the hackathons around specific topics. 

Q from fire: Do you need help making starter kits? Godot as a platform could serve as a foundation for a starter kit for projects for example. Starter kit for aframe, cryptovoxels, etc. // antpb2 offered to possibly provide hubs starter kits as well. 


msub2: Working off of a template would be acceptable imo. 

Fire: Should we form a working group to help make this? 

If the theme is around decentralization, we would feel comfortable, but if it were exclusively a blockchain hackathon it may _not_ be a good fit, as are made up of a split of preferences related to web3.

**Reply by mrmetaverse** [reply]
note: We have a hackathon section in the OMI/Angell discord, if that is useful let us know :) 

![image](https://user-images.githubusercontent.com/63426722/142256533-593dc8bd-67b0-4fbd-aa69-fbf1818f6fb0.png)

**Reply by msub2** [reply]
I'll be creating a section on the WebXR discord and submitting a call for participation for OMI later this week!

**Reply by mrmetaverse** [reply]
@msub2 ok awesome!

**Comment by mrmetaverse** [comment]


**Reply by limbofeather** [reply]
Might be better to take attendance with the usernames too
![image](https://user-images.githubusercontent.com/71227790/142256595-de6099be-9fcb-4fbb-ae2c-6aeafc4729af.png)

**Reply by mrmetaverse** [reply]
good catch!

**Comment by mrmetaverse** [comment]
Discussed [Avacon](https://www.avacon.org/blog/), there is a panel Mal is running at noon (Timezone?) on December 11th. 

What it is: 
When it is: 
Where it is happening: 
Who is going?: 
Why you should care:

**Reply by mrmetaverse** [reply]
@LightLodges can I get help filling this in? :point_up: :)

**Comment by mrmetaverse** [comment]
Global XR conference Dec 1-3. 
and a handful of other conferences coming up. 

If you have / know of an event coming up, drop them in chat below!

**Comment by mrmetaverse** [comment]
>the gltf we used for the booth should use the Omi audio emitter extension

**Comment by mrmetaverse** [comment]
next week: lets discuss leveraging roles that match up with working groups - per @humbletim 

next week: lets pick up discussion on omi-chan - per @technobaboo 

next week: move up :arrow_up: VKET discussion, as we ran out of time. Koolala said if you have any questions in the mean time, please reach out.
